### PR TITLE
RFR: Docker push/pull image actions.

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

File: packs/docker/actions/lib/docker_wrapper.py [hunk 1]
```diff
@@ -52,6 +52,24 @@ def build(self, path=None, fileobj=None, tag=None):
...
         except StopIteration:
             pass
-        except Exception, e:
+        except Exception as e:
+            sys.stderr.write('Error: %s' % (str(e)))
+            raise e
+
+    def push(self, repo, tag=None, insecure_registry=False):
+        try:
+            for line in self._client.push(repo, tag=tag, insecure_registry=insecure_registry,
```
Comment (Kami) [inline packs/docker/actions/lib/docker_wrapper.py]: Yeah, streaming is good, now we just need to implement action execution stdout and stderr streaming API ourselves :P

Comment (lakshmi-kannan) [inline packs/docker/actions/lib/docker_wrapper.py]: Yep.

File: packs/docker/actions/pull_image.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: pull_image
+runner_type: run-python
+description: Build docker image action. Equivalent to docker build.
```
Comment (Kami) [inline packs/docker/actions/pull_image.yaml]: This needs to be updated.

**Comment by Kami** [comment]
Readme also needs to be updated.

**Pull request merged**